src/coreclr/jit/emitarm64.cpp

```diff
@@ -12995,7 +12995,7 @@ void emitter::emitDispIns(
                 emitDispVectorReg(id->idReg1(), id->idInsOpt(), true);
                 emitDispVectorReg(id->idReg2(), id->idInsOpt(), false);
             }
-            if (ins == INS_cmeq)
+            if (ins == INS_cmeq || ins == INS_cmge || ins == INS_cmgt || ins == INS_cmle || ins == INS_cmlt)
```

Comment:
I wonder if it's worth a small helper to cover these 5 instructions since they get grouped together a few times

Reply:
FYI, this is the jit coding convention around subexpressions (including Boolean expressions)
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/clr-jit-coding-conventions.md#11.1

> There should be parentheses around all unary and binary expressions if they are contained within other expressions.